tests/utilities_t/test_utils.py

```diff
@@ -63,3 +71,31 @@ def test_htcondor_csv_parser(self):
                 Test2=None,
             ),
         )
+
+
+class TestSlurmCMDOptionFormatter(TestCase):
+    def test_slurm_cmd_option_formatter(self):
```

Comment:
Please also test for the exception case.

Reply:
It is already tested in https://github.com/MatterMiners/tardis/blob/db3066a6a25f793bd8e5bb59d8db430fcfaa8af6/tests/utilities_t/test_utils.py#L78-L79.